### PR TITLE
Fizzbuzz tree

### PR DESCRIPTION
**Pull request by mattburger**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401Challenges/src/main/java/code401Challenges/fizzbuzzTree/FizzBuzzTree.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package code401Challenges.fizzbuzzTree;
+
+import code401Challenges.tree.BinaryTree;
+import code401Challenges.tree.TreeNode;
+
+public class FizzBuzzTree extends BinaryTree {
```
Comment (evanslaton) [inline code401Challenges/src/main/java/code401Challenges/fizzbuzzTree/FizzBuzzTree.java]: Awesome! I haven't seen anyone else do this but it makes so much sense!
